template/tinyvue/src/views/form/advance/process-grid/index.vue

```diff
@@ -0,0 +1,221 @@
+      <tiny-grid-column
+        :title="$t('advanceForm.form.process.department')"
+        field="department"
+        :show-icon="false"
+        :editor="{
+          component: TinySelect,
+          autoselect: true,
+          attrs: {
+            options: options.department,
+          },
+        }"
+      >
+        <template #default="data">
+          <selectRender
+            :data="data"
+            :options="options.department"
+            field="department"
+          ></selectRender>
+        </template>
+      </tiny-grid-column>
+      <tiny-grid-column
+        :title="$t('advanceForm.form.process.status')"
+        field="status"
+        :show-icon="false"
+        :editor="{
+          component: TinySelect,
+          autoselect: true,
+          attrs: {
+            options: options.status,
+          },
+        }"
+      >
+        <template #default="data">
+          <selectRender
+            :data="data"
+            :options="options.status"
+            field="status"
+          ></selectRender>
+        </template>
+      </tiny-grid-column>
+      <tiny-grid-column
+        :title="$t('advanceForm.form.process.runningStatus')"
+        field="runningStatus"
+        :show-icon="false"
+        :editor="{
+          component: TinySelect,
+          autoselect: true,
+          attrs: {
+            options: options.status,
+          },
+        }"
+        :renderer="{ component: statusRender }"
+      >
+      </tiny-grid-column>
+      <tiny-grid-column
+        :title="$t('advanceForm.form.process.createTime')"
+        field="createTime"
+        :show-icon="false"
+        :editor="{
+          component: TinyDatePicker,
+          autoselect: true,
+          attrs: {
+            type: 'datetime',
+          },
+        }"
+        :renderer="defaultRender"
+      ></tiny-grid-column>
+      <tiny-grid-column
+        :title="$t('advanceForm.form.process.operation')"
+        field="operation"
+      >
+        <template #default="data">
+          <a
+            v-if="$refs.gridRef && $refs.gridRef.hasActiveRow(data.row)"
+            class="mr-2"
+            @click="saveRow(data.row)"
+          >
+            <IconSave class="operation-icon"></IconSave
+            >{{ $t('advanceForm.form.process.save') }}
+          </a>
+          <tiny-popconfirm
+            :title="$t('advanceForm.form.delete.title')"
+            type="warning"
+            trigger="click"
+            @confirm="deleteRow(data.row)"
+          >
+            <template #reference>
+              <a class="operation">
+                <IconDel class="operation-icon"></IconDel
+                >{{ $t('advanceForm.form.process.delete') }}
+              </a>
+            </template>
+          </tiny-popconfirm>
+        </template>
+      </tiny-grid-column>
+      <template #empty>
+        <span>{{ $t('advanceForm.form.nodata') }}</span>
+      </template>
+    </tiny-grid>
+  </div>
+</template>
+<script setup>
```

Comment:
建议加上 `lang="ts"`，获得更好的 TypeScript 类型提示
```suggestion
<script setup lang="ts">
```